docs/source/install.md

```diff
@@ -0,0 +1,120 @@
+# Install Fabric and Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
```

Comment:
I have a few comments on this page.
1. I though we were moving away from the gopath, and that it was not required.
2. I thought that the install script cloned the samples for you, and that doing that separately was not required.
3. The steps in this doc seem the be the same for all platforms. 
My thinking would be to to revert this doc to something closer to the current [Install](https://hyperledger-fabric.readthedocs.io/en/latest/install.html) topic and merge, since the rest is a large improvement.

Reply:
Thanks Nik, good comments, I've addressed in the latest push.